lib/facter/resolvers/freebsd/networking_resolver.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module Facter
+  module Resolvers
+    module Freebsd
+      class Networking < BaseResolver
```

Comment:
This is the networking resolver for OSX. Could you please move it to lib/facter/resolvers/networking.rb and delete the Freebsd module. Leave only Facter::Resolvers::Networking. Probably it's best to move the one from the resolvers/osx and benefit from the IDE's refactoring capabilities, because this change will break all OSX networking facts.

Reply:
Gotcha, `lib/facter/resolvers/windows/networking_resolver.rb` already defines `Facter::Resolvers::Networking`… I'll rename this `Facter::Resolvers::Windows::Networking` for consistency!